src/features/ajax/aggregate/gql.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @typedef {object} GQLMetadata
+ * @property {string} operationName Name of the operation
+ * @property {string} operationType Type of the operation
+ * @property {string} operationFramework Framework responsible for the operation
+ */
+
+/**
+ * @param {string|Object} gql The GraphQL object body sent to a GQL server
+ * @returns {GQLMetadata}
+ */
+export function parseGQL (gql) {
+  if (!gql) return
+  const contents = gql
+  if (typeof contents !== 'object' || !contents.query || typeof contents.query !== 'string') return
```

Comment:
Can simplify this.

```suggestion
  if (typeof contents !== 'object' || typeof contents.query !== 'string') return
```

Reply:
little faster to early return here if the string is empty, but little consequence either way :+1: